recipes/powershell/assets/README.md

```diff
@@ -10,13 +11,18 @@ The samples are provided only for reference and not meant for production use.
 
 Prerequisites:
 
-- NetBackup 8.3 or higher
-- PowerShell 4.0 or higher
+-NetBackup 8.3 or higher
+-PowerShell 4.0 or higher
 
-The script uses the NetBackup Asset Service API to get the VMware workload assets (filtered by the given filter if specified). 
+get_vmware_assets script uses the NetBackup Asset Service API to get the VMware workload assets (filtered by the given filter if specified). 
 It prints the details such as asset display name, instance Id, vCenter and the plan names that the asset is protected by. 
 Note: assetsFilter (should be in OData format; refer to the NetBackup API documentation) can be used to filter the assets returned. 
 It is optional; if not specified the script will print all the VM assets. Redirect the script output to a file to avoid printing the details on terminal.
 
-Use the following command to run the PowerShell sample:
+create_vmware_asset_group script creates a VMware asset group using sample data provided inside the script.  Redirect the script output to a file to 
+avoid printing the details on terminal.
+
+
+Use the following commands to run the PowerShell sample:
 .\get_vmware_assets.ps1 -MasterServer <masterServer> -username <username> -password <password> [-domainName <domainName>] [-domainType <domainType>] [-assetType <vm|vmGroup>] [-assetsFilter <filter>]
+.\create_vmware_asset_group.ps1 -MasterServer <masterServer> -username <username> -password <password> [-domainName <domainName>] [-domainType <domainType>]

```

Comment:
This appears in the same paragraph as the previous line. You should format these two commands as a list.